code/src/components/Main.js

```diff
@@ -0,0 +1,92 @@
+/* eslint-disable no-underscore-dangle */
+import React, { useState, useEffect } from 'react';
+import GetThoughts from './GetThoughts';
+import UserInput from './UserInput';
+
+const Main = () => {
+  const [getThoughts, setGetThoughts] = useState([]);
+  const [newThought, setNewThought] = useState('');
+  /* const [loading, setLoading] = useState(false); */
+
+  const APIurl = 'https://happy-thoughts-technigo.herokuapp.com/thoughts'
+
+  const fetchAPI = () => {
+    /* setLoading(true) */
+    fetch(APIurl)
+      .then((res) => res.json())
+      .then((data) => setGetThoughts(data))
+      .catch((error) => console.error(error))
+      .finally(() => console.log('no errors')/* setLoading(false) */)
+  }
+
+  useEffect(() => {
+    fetchAPI()
+    /*   const interval = setInterval(() => {
+        fetchAPI()
+      }, 60000)
+      // This line is clearing the interval when user f.ex. changes window or exit app
+      return () => clearInterval(interval) */
```

Comment:
I don't know why you commented out the Interval here but I also followed the same and it worked for me.